default.nix

```diff
@@ -0,0 +1,3 @@
+(import (fetchTarball https://github.com/edolstra/flake-compat/archive/master.tar.gz) {
```

Comment:
This is for non flake systems right? You should explain how to incorporate it without flakes

Reply:
I'm not a NixOS wizard but as far as I understand it, it reads the flake.lock and creates the hashes and uses the fetch function to download the falke stuff. so this basically removes all the flake stuff to make it compatible with non flake systems. 

to thus line of code:
it fetches a tar ball containing flake-compat. this is how you fetch stuff in legacy nix